### PR TITLE
AO3-4105 Remove "delete" button for series in works edit form

### PR DESCRIPTION
**Pull request by pixelred**
## Issue

https://otwarchive.atlassian.net/browse/AO3-4105

## Purpose

in the work edit form there was a button under the series section that just said "delete" and it would delete an entire series. if a user wants to delete an entire series, they should do it from the series edit page, so that button's been removed. 

beside it was a button that just said "remove" and it removes the work you're currently editing from the series it's in. "remove" can be a bit vague because there's no confirmation when you click it, so it's been relabelled "remove work from series" which is a bit longer; but hopefully it clears up any questions about what the button does

## Testing

confirm that the mysterious "delete" button in the series section of the work edit form is gone and the "remove" button is relabelled to "remove work from series" and working as it should be 👍

## References

related issue https://otwarchive.atlassian.net/browse/AO3-3448

### CODE REVIEW

File: app/views/works/_standard_form.html.erb [hunk 1]
```diff
@@ -185,12 +185,7 @@
                       <%= link_to serial.series.title, serial.series %>
                     </li>
                     <li>
-                      <%= link_to ts('Remove'), serial_work_path(serial),
-                              method: :delete %>
-                    </li>
-                    <li>
-                      <%= link_to ts('Delete'), serial.series,
-                              confirm: ts('Delete series from all works?'),
+                      <%= link_to ts('Remove work from series'), serial_work_path(serial),
```
Comment (sarken) [inline app/views/works/_standard_form.html.erb]: We use title case for our links in most cases, so this should be "Remove Work From Series" (we also use double, not single, quotes). 

Other than that, looks good!

**Comment by pixelred** [comment]
okay! i pushed the changes concerning this link. should i go ahead and change the single quote strings throughout the file?

**Comment by sarken** [comment]
If you want to do that, sure, I'm more than happy to review it!

**Comment by pixelred** [comment]
will do ✌️

**Comment by pixelred** [comment]
done 🔆

File: app/views/works/_standard_form.html.erb [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
+    <legend><%= ts("Preface") %></legend>
+    <h3 class="landmark heading"><%= ts("Preface") %></h3>
     <dl>
       <dt class="title required">
-        <%= f.label :title, ts('Work Title') + "*" %>
+        <%= f.label :title, ts("Work Title") + "*" %>
       </dt>
       <dd class="title required">
         <%= f.text_field :title, class: "observe_textlength text" %>
         <%= live_validation_for_field('work_title',
```
Comment (sarken) [inline app/views/works/_standard_form.html.erb]: Could you change the quotes around `work_title` here?

File: app/views/works/_standard_form.html.erb [hunk 3]
```diff
@@ -27,7 +27,7 @@
       <%= render 'pseuds/byline', form: f, type: 'work' %>
```
Comment (sarken) [inline app/views/works/_standard_form.html.erb]: Could you change the quotes on `pseuds/byline` and `work` here?

File: app/views/works/_standard_form.html.erb [hunk 4]
```diff
@@ -27,7 +27,7 @@
...
 
       <dt class="summary">
-        <%= f.label :summary, ts('Summary', max: ArchiveConfig.SUMMARY_MAX.to_s) %>
+        <%= f.label :summary, ts("Summary", max: ArchiveConfig.SUMMARY_MAX.to_s) %>
       </dt>
       <dd class="summary">
         <%= f.text_area :summary, class: "observe_textlength" %>
```
Comment (sarken) [inline app/views/works/_standard_form.html.erb]: On the line below this (line 34), `work_summary` could use double quotes.

File: app/views/works/_standard_form.html.erb [hunk 5]
```diff
@@ -94,7 +94,7 @@
         <%= check_box_tag "parent-options-show", "1", check_parent_box(@work), class: "toggle_formfield" %>
       </dt>
       <dd class="parent">
-        <%= label_tag 'parent-options-show', ts('This work is a remix, a translation, a podfic, or was inspired by another work') %>
+        <%= label_tag 'parent-options-show', ts("This work is a remix, a translation, a podfic, or was inspired by another work") %>
```
Comment (sarken) [inline app/views/works/_standard_form.html.erb]: `parent-options-show` can also have its quotes changed.

File: app/views/works/_standard_form.html.erb [hunk 6]
```diff
@@ -160,24 +160,24 @@
         <%= check_box_tag "series-options-show", "1", !@work.series.blank?, class: "toggle_formfield" %>
       </dt>
       <dd class="serial">
-        <%= label_tag 'series-options-show', ts('This work is part of a series') %>
+        <%= label_tag 'series-options-show', ts("This work is part of a series") %>
```
Comment (sarken) [inline app/views/works/_standard_form.html.erb]: `series_options_show` can get double quotes, too.

File: app/views/works/_standard_form.html.erb [hunk 7]
```diff
@@ -237,7 +237,7 @@
         <%= render 'work_form_associations_language', f: f %>
```
Comment (sarken) [inline app/views/works/_standard_form.html.erb]: Can you change the quotes on `work_form_associations_language` too?

File: app/views/works/_standard_form.html.erb [hunk 8]
```diff
@@ -246,7 +246,7 @@
                 :title,
                 { :selected => (@work.work_skin ? @work.work_skin.id.to_s : nil), include_blank: true } %>
           <p class="actions" role="navigation">
-            <%= link_to ts('Public Work Skins'), skins_path(:skin_type => 'WorkSkin') %>
+            <%= link_to ts("Public Work Skins"), skins_path(:skin_type => 'WorkSkin') %>
```
Comment (sarken) [inline app/views/works/_standard_form.html.erb]: `WorkSkin` here can also have double quotes.

File: app/views/works/_standard_form.html.erb [hunk 9]
```diff
@@ -305,17 +305,17 @@
 
       <% use_tinymce %>
         <div class="rtf-html-field">
-          <h4 class="landmark heading"><%= ts('Work Text') %>*</h4>
+          <h4 class="landmark heading"><%= ts("Work Text") %>*</h4>
           <%= c.text_area :content, value: @chapter.content,
                 class: "mce-editor observe_textlength large", id: "content" %>
           <%= live_validation_for_field('content',
```
Comment (sarken) [inline app/views/works/_standard_form.html.erb]: Can you change the quotes on `content`?

File: app/views/works/_standard_form.html.erb [hunk 10]
```diff
@@ -327,22 +327,22 @@
   <% end %><!-- end of fields for chapters -->
 
   <fieldset class="create">
-    <legend><%= ts('Post') %></legend>
+    <legend><%= ts("Post") %></legend>
     <ul class="actions">
       <% unless @work.new_record? || @work.posted? %>
         <li>
-          <%= submit_tag ts('Save Without Posting'), name: 'save_button' %>
+          <%= submit_tag ts("Save Without Posting"), name: 'save_button' %>
```
Comment (sarken) [inline app/views/works/_standard_form.html.erb]: The various `name`s for buttons down here can all have their quotation marks changes, too.

**Comment by pixelred** [comment]
okay! i think that's all of them :' )

**Comment by sarken** [comment]
I think that's all of them, too! Marking ready to merge.

**Pull request merged**